app/src/main/java/com/getsensibill/spendmanager/demo/fragment/FragmentJavaActivity.java

```diff
@@ -70,15 +54,16 @@ public void onDisplayNetworkError(boolean networkNotAvailable) {
         bundle.putBoolean(WebUiNetworkErrorFragment.ARG_NO_NETWORK_ERROR, networkNotAvailable);
         fragment.setArguments(bundle);
 
+        String TAG_WEB_NETWORK_ERROR_FRAGMENT = "TAG_WEB_NETWORK_ERROR_FRAGMENT";
         getSupportFragmentManager()
                 .beginTransaction()
-                .replace(binding.fragmentContainer.getId(), fragment, "NETWORK_ERROR_FRAGMENT_TAG")
+                .replace(binding.fragmentContainer.getId(), fragment, TAG_WEB_NETWORK_ERROR_FRAGMENT)
```

Comment:
Why aren't we using `R.id` here?

Reply:
fixed. it was like that before. updated it to be more consistent with the others